conn-sdk-cli/readmegen/templates/parameters.yaml.tmpl

```
@@ -11,6 +11,7 @@ pipelines:
         {{- range $param := .parameters }}
           {{ formatCommentYAML $param.description 10 }}
           # Type: {{ $param.type }}
+          # Required: {{- if isParameterRequired $param }} yes{{ else }} no{{ end }}
```

The blank spaces caught my eye and got me wondering why we need them, here's my suggestion to make it clearer what gets produced here (it doesn't change the output):
```suggestion
          # Required: {{ if isParameterRequired $param }}yes{{ else }}no{{ end }}
```

---

Ah, forgot to change this `- if` too.:)